django_fido/management/commands/download_authenticator_metadata.py

```diff
@@ -1,22 +1,77 @@
 """Command to download metadata for authenticators."""
 from __future__ import unicode_literals
 
+import base64
+import hashlib
 import json
-from base64 import urlsafe_b64decode
-from typing import Any, Dict
+from typing import Any, Dict, Tuple
 
 import requests
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
 from django.core.management.base import BaseCommand, CommandError
 from jwcrypto.jwk import JWK
 from jwcrypto.jws import InvalidJWSSignature
 from jwcrypto.jwt import JWT
+from OpenSSL import crypto
 
 from django_fido.constants import PEM_CERT_TEMPLATE
 from django_fido.models import AuthenticatorMetadata
 from django_fido.settings import SETTINGS
 
 
-def _get_metadata() -> Dict[str, Any]:
+class InvalidCert(Exception):
+    """Raised when certificate validation fails."""
+
+
+def urlsafe_b64decode(decodable: bytes) -> bytes:
+    """Alternative implementation to fill the necessary padding."""
+    m = len(decodable) % 4
+    if m == 2:
+        decodable += b'=='
+    elif m == 3:
+        decodable += b'='
+    return base64.urlsafe_b64decode(decodable)
+
+
+def verify_certificate(jwt: JWT) -> JWK:
+    """Get (and verify) the signing key from JWT."""
+    # First element in the header is our actual key
+    try:
+        decoding_key = JWK.from_pem(PEM_CERT_TEMPLATE.format(jwt.token.jose_header['x5c'][0]).encode())
+    except ValueError:
+        raise InvalidCert('Cannot decode key.')
+    if SETTINGS.metadata_service['certificate'] is None:
+        return decoding_key
```

Comment:
Disabling verification should be done the same way as in metadata.

Reply:
This is from #70, will go away after a rebase.